src/Polytopes/Polyhedron/properties.jl

```diff
@@ -87,11 +107,34 @@ Base.length(iter::VertexPointIterator) = nvertices(iter.p)
 """
    vertices(P::Polyhedron, [,as::Type{T} = Points])
```

Comment:
have you considered `vertices(::Type{bla}, P)` as an alternative to `(P, as = bla)`? Julia odes this for `round`, `floor`, ...

Reply:
Agreed, and we also do this elsewhere in Oscar (e.g. for `order` of groups, where one specify the desired integer type for the return value). So the caller would write `vertices(Points, P)`. And you could still allow `vertices(P)` by providing a convenience method to set the default:  `vertices(P::Polyhedron) = vertices(Points, P)`

Reply:
Here the first argument is not the type of the return value, so how does this fit with the functions you mention?